tutorials/eth-api/using-tenderly.md

```diff
@@ -0,0 +1,175 @@
+
+![Create a Tenderly account](/images/tutorials/eth-api/using-tenderly/tenderly-1.png)
+
+There is a limit of one project with a free account, however, you can have multiple smart contracts under the purview of a single project.
+
+## Add Smart Contracts {: #add-smart-contracts }
+
+Adding a smart contract to your Tenderly project is akin to bookmarking it. While not required, adding a contract will unlock additional Tenderly features over simply searching for the contract in the Tenderly platform. 
+
+To add a smart contract to your Tenderly project, Click on the **Contracts** tab under the Inspect heading, then Click **Add Contracts**. Then, take the following steps:
+
+1. Enter the address of the contract. For this tutorial, we'll be using the Multichain USDC contract `0x818ec0a7fe18ff94269904fced6ae3dae6d6dc0b` 
+2. Select the network the contract is deployed to. We'll select Moonbeam in this case
+3. Give the contract a name to help you recognize it in the dashboard
+4. Press **Add Contract** 
+
+![Add a smart contract](/images/tutorials/eth-api/using-tenderly/tenderly-2.png)
+
+## Simulate a Transaction {: #simulate-a-transaction }
+
+Simulations allow you to see how a transaction will execute without actually sending it to execute on the blockchain. You can simulate a transaction against any point in time, or simply the latest block. 
+
+Let's craft a transaction to simulate against the Moonbeam network by taking the following steps: 
+
+1. Select the contract that you'd like to interact with. The name displayed here is the nickname that you gave the contract when [adding it to your Tenderly workspace](#add-smart-contracts) 
+2. Select the contract function you'd like to call. `Transfer` is selected for demonstration purposes
+3. Next, we'll input relevant function parameters. For destination address you can input any address, such as Alith's address: `0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac` 
+4. For amount, you can also specify any amount, such as `10000000000`
+5. Select **Pending Block** to run the simulation against the latest Moonbeam block produced 
+6. Specify the from address as Baltathar: `0x3Cd0A705a2DC65e5b1E1205896BaA2be8A07c6e0` or another address of your choice
+7. Press **Simulate Transaction**
+
+
+![Simulate a transaction against Moonbeam](/images/tutorials/eth-api/using-tenderly/tenderly-3.png)
+
+Clearly, this simulated transaction is going to fail because we're trying to send 10,000 USDC that we don't have. But, with [Tenderly simulator](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}, we can tinker with the blockchain state and run simulations that assume different conditions. For example, let's run the simulation assuming that Baltathar actually holds a balance of 10,000 USDC. To do so, take the following steps: 
```

Comment:
might want to add that the user will need to click Re-Simulate in the top right corner